python/pythonmonkey/require.py

```diff
@@ -85,8 +85,8 @@
 # Python 3.13 dramatically changed how the namespace in `exec`/`eval` works
 # See https://docs.python.org/3.13/whatsnew/3.13.html#defined-mutation-semantics-for-locals
 _locals = {}  # keep the local variables inside `eval`/`exec` to a dict
-globalThis.python.eval = lambda x: eval(x, None, _locals)
-globalThis.python.exec = lambda x: exec(x, None, _locals)
+globalThis.python.eval = lambda x: eval(str(x)[:], None, _locals)
+globalThis.python.exec = lambda x: exec(str(x)[:], None, _locals)
```

Comment:
> this create copies of strings passed to this function

This is actually intended.

In `tests/python/test_reentrance_smoke.py:19`, it sometimes errors with `Error: Python SyntaxError: source code string cannot contain null bytes` in Python debug build because of the changes in https://github.com/Distributive-Network/PythonMonkey/pull/443/commits/73957c02ef459ca28739059c3847bbc698998283